packages/dbml-cli/package.json

```diff
@@ -1,6 +1,6 @@
 {
```

Comment:
Please also add `snowflake` option to the sql2dbml CLI (and some test in dbml-cli). You can check this https://github.com/holistics/dbml/pull/486 for an example on how to do that